yarn.lock

```diff
@@ -1955,6 +1955,23 @@
     "@griffel/react" "^1.5.14"
     "@swc/helpers" "^0.5.1"
 
+"@fluentui/tokens@1.0.0-alpha.18":
```

Comment:
this shouldn't exist here. 

if your package will depend on monorepo workspace package it need to depent on the same version as specified here 

https://github.com/microsoft/fluentui/blob/master/packages/web-components/package.json#L4

you'll need to adjust releases configuration to cross bump this your packages everytime dependant ones release occured

Reply:
Yes. These are unintentional. Came via a master merge. Reverted and adjusted the dependency ranges.